constants/src/lib.rs

```diff
@@ -62,6 +64,12 @@ pub type CurvePoint = GenericCurvePoint<SystemCurveGroup>;
 #[cfg(feature = "mpc-types")]
 pub type AuthenticatedScalar = AuthenticatedScalarResult<SystemCurveGroup>;
 
+/// The (unbatched) proof type that the proof system generates
+pub type SystemProof = Proof<SystemCurve>;
+
+/// The polynomial commitment type that the proof system utilizes
+pub type SystemCommitment = Commitment<SystemCurve>;
```

Comment:
Same here, does not belong in constants IMO, `circuit-types` may be a better spot. Also would not name these `SystemCommitment` as they do not parameterize the system. Maybe `PlonkProof` (analog of `PlonkCircuit` already defined in `circuits`) and `PolynomialCommitment` to give specificity